src/map.h

```diff
@@ -310,8 +310,8 @@ class Map /*: public NodeContainer*/
 	// This stores the properties of the nodes on the map.
 	const NodeDefManager *m_nodedef;
 
-	bool isOccluded(v3s16 p0, v3s16 p1, float step, float stepfac,
-			float start_off, float end_off, u32 needed_count);
+	bool isOccluded(v3s16 pos_camera, v3s16 pos_blockcenter, float step,
+		float stepfac, float start_offset, float end_offset, u32 needed_count);
```

Comment:
Inconsistent naming: the declaration uses `start_offset` while the definition (incl. the body) uses `offset`. `offset` makes the code look cleaner, but `start_offset` is easier to understand.

Reply:
Intended. `isOccluded` just recycles the variable, thus no renaming there.